### DISCUSSION TITLE
Switch to dart sass

### DISCUSSION THREAD

**Discussion by danemorgan** [holger1411/understrap#1357] [Ideas]
Should Underscores be moving to dart sass using @use and @forward?

https://sass-lang.com/dart-sass

**Comment by 0dp** [comment]
Understrap you mean?
Yes, Ian already made a PR that you merged into develop branch that uses dart instead.

But changing to use and forward might not be the best move since bootstrap is not doing it

**Reply by danemorgan** [reply]
We had a user make that request, but I agree, sometimes the answer should be no, and conforming to the way Bootstrap does it is a good reason for no in my mind.

**Reply by 0dp** [reply]
Now when underscores is on the table. Isn't it time to just copy the few needed bits from underscores into the theme and drop the underscores dependency (if not already)

**Reply by danemorgan** [reply]
I suspect Underscores is a vestigial limb at this point. But the name familiarity demands we wave that limb about the place. 👻
